docs/platforms/flutter/features/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Features
+description: "Learn more about Sentry's Flutter SDK features."
+sidebar_order: 1
+---
+
+<Note>
+
+Sentry's Flutter SDK enables automatic reporting of errors, messages, and exceptions. The SDK is available on GitHub [`sentry-dart`](https://github.com/getsentry/sentry-dart/).
+
+</Note>
+
+**Features:**
+
+- Automatic native crash error tracking (using both [Android](/platforms/android/) and [iOS](/platforms/apple/)), including
+    - Java, Kotlin, C, and C++ code for Android
+    - ObjC, Swift, and C for iOS
+- Offline storage of events
+- Events [enriched](/platforms/flutter/enriching-events/context/) with device data
+- Breadcrumbs automatically captured:
+    - by the [Flutter SDK](/platforms/flutter/enriching-events/breadcrumbs/#automatic-breadcrumbs)
+    - via the Native SDKs [Automatic Breadcrumbs for Android](/platforms/android/enriching-events/breadcrumbs/#automatic-breadcrumbs) and [Automatic Breadcrumbs for iOS](/platforms/apple/usage/#sentryautobreadcrumbtrackingintegration)
+    - as well as `http` with the [Dart SDK](/platforms/dart/)
```

Comment:
Shouldn't we keep this list specific to the Flutter SDK features? Not sure how true this is, but the Dart features list seem to be more scoped to just Dart in comparison

Reply:
Good question - the way I see it, Dart is a subset of Flutter, so it can make sense to list all Dart features here as well - or we point users to the Dart features page. Both is fine for me 👍 
@buenaflor wdyt? Also please give this page a close review as the info might be outdated

Reply:
I think we can definitely point to the Dart page saying that all the Dart features are also included automatically

Reply:
One thing I'm not sure yet is when we add a new integration should we also update the features list to include that? Or maybe just point to the specific page with all the integrations?

like here: https://docs.sentry.io/platforms/flutter/integrations/

Reply:
IMO the Features page is a good way to get a quick overview of features and what the SDK supports - and also for us to call out new features. It doesn't need to be 100% complete (e.g. don't need to specifically list every single integration), we can decide case-bya-case. 
We can add a bullet point below specific integrations along the lines of "For a complete list, see our Integrations page"